pkgs/dart_flutter_team_lints/lib/analysis_options.yaml

```diff
@@ -17,7 +17,38 @@ include: package:lints/recommended.yaml
 
```

Comment:
Any desire to add the "language" section here and at least do `strict-casts`? @devoncarew ?

Reply:
Here's the three strict options:

```
    strict-casts: true
    strict-inference: true
    strict-raw-types: true
```

cc @natebosch, but I'd wonder if these are things we generally want turned on, and, how they'd land w/ our current repos (would the settings cause work to update the repos).

Reply:
strict-casts is almost always goodness, AFAIC – the others can be a bit noisy

Reply:
Yes, I don't see any reason we wouldn't want to turn on `strict-casts` everywhere.